it/mail/chrome/messenger/am-addressing.dtd

```diff
@@ -1,54 +1,54 @@
 <!-- extracted from am-addressing.xul -->
 
-<!ENTITY addressing.label          "Composizione ed indirizzi">
+<!ENTITY addressing.label          "Composizione e indirizzi">
 <!ENTITY addressingGroupTitle.label "Indirizzi">
-<!ENTITY addressingText.label      "Quando si cercano indirizzi:">
+<!ENTITY addressingText.label      "Durante la ricerca di indirizzi:">
 <!-- LOCALIZATION NOTE (override.label) : do not translate "LDAP" in below line -->
-<!ENTITY useGlobal.label           "Utilizza le preferenze globali per i server LDAP per questo account">
+<!ENTITY useGlobal.label           "Utilizza le mie preferenze globali server LDAP per questo account">
 <!ENTITY useGlobal.accesskey       "U">
 <!ENTITY editDirectories.label     "Modifica cartelle…">
 <!ENTITY editDirectories.accesskey "M">
 <!-- LOCALIZATION NOTE (directories.label) : do not translate "LDAP" in below line -->
-<!ENTITY directories.label         "Utilizza un diverso server LDAP:">
+<!ENTITY directories.label         "Utilizza un server LDAP differente:">
 <!ENTITY directories.accesskey     "d">
 
 <!-- am-addressing.xul -->
 
 <!ENTITY compositionGroupTitle.label "Composizione">
 <!-- LOCALIZATION NOTE (useHtml.label) : do not translate "html" in below line -->
-<!ENTITY useHtml.label                        "Componi i messaggi in HTML">
+<!ENTITY useHtml.label                        "Componi i messaggi in formato HTML">
 <!ENTITY useHtml.accesskey                    "C">
-<!ENTITY autoQuote.label                      "Inserisci automaticamente il messaggio originale nella risposta">
+<!ENTITY autoQuote.label                      "Cita automaticamente il messaggio originale nella risposta">
 <!ENTITY autoQuote.accesskey                  "I">
 <!-- LOCALIZATION NOTE (then.label): This will concatenate with the 4 strings that follow. -->
-<!ENTITY then.label                           "dopodiché">
+<!ENTITY then.label                           "In seguito,">
 <!ENTITY then.accesskey                       "o">
-<!ENTITY aboveQuote.label                     "inizia la risposta sopra il testo a cui si risponde">
-<!ENTITY belowQuote.label                     "inizia la risposta sotto il testo a cui si risponde">
-<!ENTITY selectAndQuote.label                 "seleziona il testo a cui si risponde">
-<!ENTITY place.label                          "e metti la firma">
+<!ENTITY aboveQuote.label                     "inizia la mia risposta sopra il testo citato">
```

Comment:
CORRECTIONS NEEDED. 

Here the possessive pronoun "mia" is redundant. Moreover, it's not used in the next string (#27), so I would avoid "mia" also to improve the overall consistency of the text.